pkg/controller/account/account_controller.go

```diff
@@ -67,6 +67,8 @@ const (
 	iamUserNameSRE = "osdManagedAdminSRE"
 
 	controllerName = "account"
+
+	maxConcurrentReconciles = 4
```

Comment:
Whoah!

This controller is *not* ready to go concurrent!

Reply:
Sorry folks, I didn't mean to drop this bomb and run, I was on my way out and wanted to express concern.

More at the top level...